backend/adapter_processor/serializers.py

```diff
@@ -63,9 +64,10 @@ def to_representation(self, instance: AdapterInstance) -> dict[str, str]:
 
         adapter_metadata = instance.get_adapter_meta_data()
         rep[AdapterKeys.ADAPTER_METADATA] = adapter_metadata
-        adapter_metadata[AdapterKeys.ADAPTER_CONTEXT_WINDOW_SIZE] = (
-            instance.get_context_window_size()
-        )
+        if instance.adapter_type == AdapterTypes.LLM.value:
+            adapter_metadata[AdapterKeys.ADAPTER_CONTEXT_WINDOW_SIZE] = (
+                instance.get_context_window_size()
+            )
```

Comment:
@gaya3-zipstack since we have added this in an if condition this would mean that this value itself might not be added in the adapter_metadata if we don't enter this clause.  

Should we add a else case here. If it is not needed may be add a comment in the code why skipping else is okay here

Reply:
Context window is only relevant for LLMs @ritwik-g . Added a comment.